src/GuildProfile/components/GuildProfileModal/GuildRoles.jsx

```diff
@@ -0,0 +1,64 @@
+/* @license
+ * Copyright (c) 2020 NurMarvin (Marvin Witt)
+ * Copyright (c) 2021 jaimeadf (Jaime Filho)
+ * Licensed under the Open Software License version 3.0
+ */
+
+import React from 'react';
+
+import { WebpackModules } from '@zlibrary/api';
+
+import { useStateFromStores } from '@discord/Flux';
+import i18n from '@discord/i18n';
+import Text from '@discord/components/Text';
+import Flex from '@discord/components/Flex';
+
+import { ScrollerThin } from '@discord/components/Scroller';
+
+const StreamerModeStore = WebpackModules.getByProps('hidePersonalInformation');
+
+const classes = {
+    margins: {
+        ...WebpackModules.getByProps('marginBottom8')
+    },
+    list: WebpackModules.getByProps('empty', 'emptyIconStreamerMode', 'emptyText'),
+    infoSection: WebpackModules.getByProps('infoScroller'),
+    row: WebpackModules.getByProps('listRow'),
+    roleTag: WebpackModules.getByProps('roleTag'),
+    role: WebpackModules.getByProps('role')
+};
+const baseRoleColor = '#fff';
```

Comment:
It would be good to use the color found on `DiscordConstants.HEXColors.PRIMARY_DARK_300` which is the same discord uses for roles.